### DISCUSSION TITLE
Participating Media is not working

### DISCUSSION THREAD

**Discussion by Ghiraa** [mitsuba-renderer/mitsuba3#1063] [Q&A]
Hello, I am trying to fill a box with water to see how light scatters. Although in the documentation there is an example of how to add an interior attribute to an obj shape (https://mitsuba.readthedocs.io/en/latest/src/generated/plugins_media.html):
![image](https://github.com/mitsuba-renderer/mitsuba3/assets/45585468/46cbf21f-a83f-45f0-b72f-0285e40cebb9)
Mine is not working, resulting in this error:
![image](https://github.com/mitsuba-renderer/mitsuba3/assets/45585468/908dd4a2-f12a-4467-b5dd-9f5b3f52df27)

This is my code:
![image](https://github.com/mitsuba-renderer/mitsuba3/assets/45585468/1272d00c-af50-41cd-bfb0-111ae8fbe1ad)

Thank you!

**Comment by ziyi-zhang** [comment]
Hi, the value of key `interior` is the string "water", not the homogeneous medium defined above.
You can for instance do
```
water = {
  'type': 'homogenous',
  ....
}

'box': {
    ...
   'interior': water
}
```

**Reply by Ghiraa** [reply]
Hi, if I do this it complains that water is not defined.

![image](https://github.com/mitsuba-renderer/mitsuba3/assets/45585468/2ebb8108-0db0-4d37-96d8-2ea9584d6b81)

**Reply by Ghiraa** [reply]
nevermind, I solved the issue
